### ISSUE TITLE
Rename VIIRS AOD obs to be in line with the rest of the dump

### ISSUE THREAD

**Issue by CoryMartin-NOAA** [NOAA-EMC/global-workflow#3022]
### Machine

Hera/Ursa

### Start date

2021-12-20

### End date

2021-12-21

### Additional information

I'm trying to make the aerosol dump files follow convention rather than being unique, but that will require the CI tests to have the sufficiently renamed files staged.
Can you copy/symlink the following from:
```
gdas.20211220/18/atmos/gdas.t18z.viirs_n20.2021122018.nc4
gdas.20211220/18/atmos/gdas.t18z.viirs_npp.2021122018.nc4
gfs.20211220/18/atmos/gfs.t18z.viirs_n20.2021122018.nc4
gfs.20211220/18/atmos/gfs.t18z.viirs_npp.2021122018.nc4
gdas.20211221/00/atmos/gdas.t00z.viirs_n20.2021122100.nc4
gdas.20211221/00/atmos/gdas.t00z.viirs_npp.2021122100.nc4
gfs.20211221/00/atmos/gfs.t00z.viirs_n20.2021122100.nc4
gfs.20211221/00/atmos/gfs.t00z.viirs_npp.2021122100.nc4
```
to:
```
gdas.20211220/18/atmos/gdas.t18z.viirs_n20_aod.tm00.nc
gdas.20211220/18/atmos/gdas.t18z.viirs_npp_aod.tm00.nc
gfs.20211220/18/atmos/gfs.t18z.viirs_n20_aod.tm00.nc
gfs.20211220/18/atmos/gfs.t18z.viirs_npp_aod.tm00.nc
gdas.20211221/00/atmos/gdas.t00z.viirs_n20_aod.tm00.nc
gdas.20211221/00/atmos/gdas.t00z.viirs_npp_aod.tm00.nc
gfs.20211221/00/atmos/gfs.t00z.viirs_n20_aod.tm00.nc
gfs.20211221/00/atmos/gfs.t00z.viirs_npp_aod.tm00.nc
```

Please copy these to all the platforms, for CI testing, thanks!

**Comment by KateFriedman-NOAA** [comment]
@CoryMartin-NOAA I have renamed the files as requested in the GDA on Hera. Please confirm the files are good and I will sync them to the other platforms, thanks!
```
[role.glopara@hfe09 dump]$ pwd
/scratch1/NCEPDEV/global/glopara/dump
[role.glopara@hfe09 dump]$ ll g*.20211220/18/atmos/*viirs*
-rw-r--r-- 1 role.glopara global 967425 Feb 16  2024 gdas.20211220/18/atmos/gdas.t18z.viirs_n20_aod.tm00.nc
-rw-r--r-- 1 role.glopara global 967425 Feb 16  2024 gdas.20211220/18/atmos/gdas.t18z.viirs_npp_aod.tm00.nc
-rw-r--r-- 1 role.glopara global 967425 Feb 16  2024 gfs.20211220/18/atmos/gfs.t18z.viirs_n20_aod.tm00.nc
-rw-r--r-- 1 role.glopara global 967425 Feb 16  2024 gfs.20211220/18/atmos/gfs.t18z.viirs_npp_aod.tm00.nc
[role.glopara@hfe09 dump]$ ll g*.20211221/00/atmos/*viirs*
-rw-r--r-- 1 role.glopara global 917601 Feb 16  2024 gdas.20211221/00/atmos/gdas.t00z.viirs_n20_aod.tm00.nc
-rw-r--r-- 1 role.glopara global 985169 Feb 16  2024 gdas.20211221/00/atmos/gdas.t00z.viirs_npp_aod.tm00.nc
-rw-r--r-- 1 role.glopara global 917601 Feb 16  2024 gfs.20211221/00/atmos/gfs.t00z.viirs_n20_aod.tm00.nc
-rw-r--r-- 1 role.glopara global 985169 Feb 16  2024 gfs.20211221/00/atmos/gfs.t00z.viirs_npp_aod.tm00.nc
```

**Comment by CoryMartin-NOAA** [comment]
Looks good, thanks @KateFriedman-NOAA ! Note that since they are moved, I'll need to issue a quick PR to address this and nothing else or else future CI testing will fail. I'll do so ASAP.

**Comment by KateFriedman-NOAA** [comment]
Sounds good @CoryMartin-NOAA ! I can review that PR quickly. I will sync the update to the other platforms now.

**Comment by KateFriedman-NOAA** [comment]
@WalterKolczynski-NOAA please sync the `20211220` and `20211221` dump files from Hera to MSU, thanks!

**Comment by KateFriedman-NOAA** [comment]
VIIRS dump files are updated on WCOSS2 and Jet.

**Comment by WalterKolczynski-NOAA** [comment]
Requested dates synchronized to MSU

**Comment by KateFriedman-NOAA** [comment]
Thanks @WalterKolczynski-NOAA ! Sync of updated dump files is now complete. Closing as complete.

**Comment by CoryMartin-NOAA** [comment]
I've realized that the 'quick' PR won't be so simple since it would require a GDASApp hash update. We have several moving pieces there, if we need to temporarily turn off the snow/aero CI test in the future if we aren't able to update before then, we can cross that bridge then.

**Comment by KateFriedman-NOAA** [comment]
Okie dokie...let me know if you foresee it taking even longer...I can create symlinks with the old filenames if needed.

**Comment by RussTreadon-NOAA** [comment]
@KateFriedman-NOAA , @WalterKolczynski-NOAA , and @CoryMartin-NOAA

As noted above, completion of this issue breaks _C96C48_hybatmaerosnowDA_ g-w CI.  Job gdas_aeroanlinit fails on Hera with the error message
```
  File "/scratch1/NCEPDEV/da/Russ.Treadon/git/global-workflow/pr2978/ush/python/wxflow/fsutils.py", line 87, in cp
    raise OSError(f"unable to copy {source} to {target}")
OSError: unable to copy /scratch1/NCEPDEV/stmp2/Russ.Treadon/COMROOT/praero_pr2978/gdas.20211220/18/obs/gdas.t18z.viirs_n20.202112201\
8.nc4 to /scratch1/NCEPDEV/stmp2/Russ.Treadon/RUNDIRS/praero_pr2978/gdas.2021122018/gdasaeroanl_18/obs/gdas.t18z.viirs_n20.2021122018.nc4
+ JGLOBAL_AERO_ANALYSIS_INITIALIZE[1]: postamble JGLOBAL_AERO_ANALYSIS_INITIALIZE 1729629955 1
+ preamble.sh[70]: set +x
End JGLOBAL_AERO_ANALYSIS_INITIALIZE at 20:46:16 with error code 1 (time elapsed: 00:00:21)
+ aeroanlinit.sh[1]: postamble aeroanlinit.sh 1729629928 1
+ preamble.sh[70]: set +x
End aeroanlinit.sh at 20:46:16 with error code 1 (time elapsed: 00:00:48)
```

Should we add the fix to PR #2978 or open a new PR?

**Comment by KateFriedman-NOAA** [comment]
> Should we add the fix to PR https://github.com/NOAA-EMC/global-workflow/pull/2978 or open a new PR?

If #2978 either needs the fix and/or it will go in soon then I say put it in that PR. If not, let's do a separate PR ASAP.

**Comment by CoryMartin-NOAA** [comment]
we will need to include it in 2978, otherwise other things will be broken. Let me open a GDASApp PR this AM.

**Comment by RussTreadon-NOAA** [comment]
@KateFriedman-NOAA and @WalterKolczynski-NOAA:  When you closed g-w issue #3012 yesterday, you removed the remaining item to be completed before you could trigger g-w CI for PR #2978.  

I fired off DA g-w CI jobs after I updated `gdas_fv3jedi_ver` in PR #2978.  I saw the _C96C48_hybatmaerosnowDA_ failure this morning.  If PR #2978 requires all active g-w CI to pass, it's stuck until we fix this new issue.

**Comment by KateFriedman-NOAA** [comment]
@CoryMartin-NOAA @RussTreadon-NOAA Noted, so yes please include the fix in PR #2978, thanks!
